### DISCUSSION TITLE
EasyEdge PaddleOCR v4?

### DISCUSSION THREAD

**Discussion by gutenye** [PaddlePaddle/PaddleOCR#12268] [Q&A]
Only PaddleOCR v3 is available in the https://ai.baidu.com/easyedge/app/openSource?from=paddlelite 
1. Is v4 available for download?
2. If not, how can I replace the v3 demo with v4 model?

**Comment by UserWangZz** [comment]
https://github.com/PaddlePaddle/PaddleOCR#%EF%B8%8F-pp-ocr%E7%B3%BB%E5%88%97%E6%A8%A1%E5%9E%8B%E5%88%97%E8%A1%A8%E6%9B%B4%E6%96%B0%E4%B8%AD
you can download ppocrv4 model in this link.

**Comment by gutenye** [comment]
@UserWangZz 

The PP-OCRv3 iOS SDK I downloaded from  https://ai.baidu.com/easyedge/app/openSource?from=paddlelite has two files `model` and `params`

<img width="312" alt="SCR-20240506-lttt" src="https://github.com/PaddlePaddle/PaddleOCR/assets/377544/a53b746a-ca78-4258-9417-f3a8c2b24f3c">

The ppocrv4 models from your link have four files (detection and recognition)

```
ch_PP-OCRv4_det_infer/inference.pdmodel
ch_PP-OCRv4_det_infer/inference.pdiparams
ch_PP-OCRv4_rec_infer/inference.pdmodel
ch_PP-OCRv4_rec_infer/inference.pdiparams
```

How can I replace the two files (PP-OCRv3 iOS SDK) with the four files (ppocrv4)?

**Comment by tink2123** [comment]
Yes, you can directly replace the model file with ppocr_v4

**Comment by gutenye** [comment]
@tink2123 the SDK only has two files, while ppocr_v4 has four files. Which files should I use?

**Comment by HaloWang** [comment]
Same issue here.
